dropwizard-metrics-listener/pom.xml

```diff
@@ -40,6 +40,7 @@
             <groupId>${project.groupId}</groupId>
             <artifactId>pushy</artifactId>
             <version>${project.version}</version>
+            <classifier>shaded</classifier>
```

Comment:
This feels a little odd to me; I don't think we want to make the Dropwizard metrics listener explicitly dependent upon the shaded version of Pushy. The [Maven docs](https://maven.apache.org/pom.html) aren't quite clear on this point; if we leave out a classifier here, will any classification of com.turo.pushy at the same version fulfill the requirement?

Reply:
Hmm. You're right, that's a bit strange. 

What could be done, for example, is also providing a shaded `dropwizard-metrics-listener` that had the shaded `pushy` dependency. However I don't even know if that's possible with maven.

What also could be done, which I think would be easier, is marking the `pushy` dependency in `dropwizard-metrics-listener` as optional, which would force users of the artifact, to also add `pushy` as a dependency explicitly (and then they'd be able to use either the shaded or the  non-shaded one).

Reply:
From https://maven.apache.org/pom.html: ```Corresponds to the dependant artifact's packaging type. This defaults to jar```

Reply:
@jchambers From : https://maven.apache.org/guides/introduction/introduction-to-dependency-mechanism.html```NOTE: In two of these dependency references, we had to specify the <type/> element. This is because the minimal set of information for matching a dependency reference against a dependencyManagement section is actually {groupId, artifactId, type, classifier}. In many cases, these dependencies will refer to jar artifacts with no classifier. This allows us to shorthand the identity set to {groupId, artifactId}, since the default for the type field is jar, and the default classifier is null.```